### PR TITLE
Prep for CSP implementation

### PR DESCRIPTION
**Pull request by jkassemi**
Changes that will be required to address CSP violations.

* Removes inline scripts
* Removes inline styles
* Moves webpack closer to the beginning of the metalsmith build, will permit configuring fingerprinting and hashes for better script control (such as `identity` attributes on script tags, and `sha256-X` CSP declarations).
* Uses production build settings for staging. Goal to ensure we can test CSP header on staging environment without requiring eval-based source map.

Part of https://github.com/department-of-veterans-affairs/vets.gov-team/issues/611

### CODE REVIEW

File: assets/js/google-analytics.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
```
Comment (jkassemi) [inline assets/js/google-analytics.js]: Removes inline script.

File: assets/js/usa-search.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+var usasearch_config = { siteHandle:"vets.gov_search" };
```
Comment (jkassemi) [inline assets/js/usa-search.js]: Removes inline script

File: assets/js/user-voice.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+UserVoice=window.UserVoice||[];(function(){var uv=document.createElement('script');uv.type='text/javascript';uv.async=true;uv.nonce='EqfyqZFQCKLmLMcdCyziDqosi6wTGa';uv.src='https://widget.uservoice.com/NDkEKHU7ZuU64RQKkUL9Q.js';var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(uv,s)})();
```
Comment (jkassemi) [inline assets/js/user-voice.js]: Removes inline script

**Review by crwallace — APPROVED** [review]
This look great!

**Pull request merged**